internal/controller/volumereplicationgroup_controller.go

```diff
@@ -2643,3 +2657,47 @@ func (v *VRGInstance) validateSecondaryPVCConflictForVolRep() *metav1.Condition
 
 	return nil
 }
+
+func (v *VRGInstance) aggregateVRGAutoCleanupCondition() *metav1.Condition {
+	// Structured logging with key/value pairs is clearer and friendlier to log aggregation
+	v.log.Info(
```

Comment:
We already have replication state added as a K/V pair to the structured log [here](https://github.com/RamenDR/ramen/blob/8d2d4344de0eaa5a99aa5417db4069abb79bdba0/internal/controller/volumereplicationgroup_controller.go#L590), so it is not required to add it again here.

Further I would remove this log entirely it is not providing any added value as such.